### PR TITLE
docs: Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SiQube**
related to #543

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -145,7 +158,7 @@ functionality. We try to keep our code coverage close to 100%.
 It is possible to limit the scope of testing to specific environments and files. For example, to
 only test event related functionality using the Python 3.7 environment use:
```
Comment (dkrako) [inline CONTRIBUTING.md:159]: this line has to be updated too

**Review by dkrako — CHANGES_REQUESTED** [review]
Thanks a lot!

there's just a single change request, one occurence of python37 has been skipped.

File: CONTRIBUTING.md (hunk 1 above)
Comment (SiQube) [reply CONTRIBUTING.md]: good eye ! resolved!

**Comment by SiQube** [comment]
added my changes @dkrako

**Pull request merged**